runtime/src/status_cache.rs

```diff
@@ -223,27 +223,38 @@ impl<T: Serialize + Clone> StatusCache<T> {
 
     // returns the statuses for each slot in the slots provided
     pub fn slot_deltas(&self, slots: &[Slot]) -> Vec<SlotDelta<T>> {
```

Comment:
because of my another false alarm, i think we can just remove `fn slot_deltas` entirely?